### ISSUE TITLE
NTR: Mucosal thickening (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8000]
New term request Mucosal thickening (mayman_090521220853):   UMLS: 3686502

**Comment by pnrobinson** [comment]
@MickeySegal

Do we mean thickening of the sinus mucosae as one sees in sinusitis?

**Comment by MickeySegal** [comment]
In Mucolipidosis II https://www.ncbi.nlm.nih.gov/books/NBK1828/ "Progressive mucosal thickening narrows the airways".

As you point out, this would benefit from some explanation to distinguish it from the temporary inflammatory response in sinusitis.

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000099
- label: Airway mucosal thickening
- parents: [Abnormal respiratory epithelium morphology (HP:0012253)](https://hpo.jax.org/app/browse/term/HP:0012253)
- pmid: PMID:32957425
- def: Increase in thickness of the mucosa of the airways above the upper limit of normal.
- comment: This feature can be observed in mucolipidosis type II. Progressive airway thickening can lead to respiratory insufficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8000
- synonyms:
